driver/conv_driver.hpp

```diff
@@ -1423,14 +1381,25 @@ int ConvDriver<Tgpu, Tref>::AllocateBuffersAndCopy()
             b_dev->ToGPU(q, b.data.data());
             db_dev->ToGPU(q, db.data());
         }
+    }
 
-        if(!weiRead)
+    if(!dataRead)
+    {
+        for(int i = 0; i < in_sz; i++)
+        {
+            auto val = prng::gen_0_to_B(Data_scale);
```

Comment:
Please comment why we generate numbers even if we don't intend to use them.

Reply:
Done